server/chat-commands.js

```diff
@@ -1008,6 +1008,10 @@ const commands = {
 			if (!this.can('editroom', null, room)) return;
 		} else if (room.battle) {
 			if (!this.can('editprivacy', null, room)) return;
+			if (room.battle) {
```

Comment:
Same issue here (dupe conditions).

Reply:
Thanks. I think I copied this block from elsewhere when I was trying to make things more general